src/components/ShoppingList/PopulatedList.js

```diff
@@ -109,8 +123,11 @@ const PopulatedList = () => {
                     groceryItem.data().lastPurchasedDate,
                   )}
                 ></input>
-                <li onClick={() => deleteItemHandler(groceryItem.id)}>
+                <li>
                   {groceryItem.data().itemName}
+                  <button onClick={() => deleteItemHandler(groceryItem.id)}>
+                    Delete
+                  </button>
```

Comment:
I just commented out the delete handler on our end in an effort to get more ARIA compliant so I think this is great timing and will merge easily. It also looks like Sweet Alert is pretty accessibility-focused so that's very cool too.